### ISSUE TITLE
readme mistake: omxh264enc periodicity-idr -> periodicty-idr

### ISSUE THREAD

**Issue by alexeypolitov** [awslabs/amazon-kinesis-video-streams-producer-sdk-cpp#217]
In omxh264enc parameters you use `periodicity-idr`, but correct parameter name will be `periodicty-idr`
```
gst-launch-1.0 -v v4l2src do-timestamp=TRUE device=/dev/video0 ! videoconvert ! video/x-raw,format=I420,width=640,height=480,framerate=30/1 ! omxh264enc periodicity-idr=45 inline-header=FALSE ! h264parse ! video/x-h264,stream-format=avc,alignment=au ! kvssink stream-name=YourStreamName access-key="YourAccessKey" secret-key="YourSecretKey"
```

**Comment by unicornss** [comment]
Thanks @alexeypolitov for noticing the parameter name incorrectly spelled in this particular gst-launch-1.0 command.  We will update in the upcoming release. 

Thanks again.

**Comment by chehefen** [comment]
README has been corrected in the release 2.0.0.
